iquip/apps/explorer.py

```diff
@@ -161,6 +159,7 @@ def lazyLoadFile(self, experimentFileItem: QTreeWidgetItem):
             self._addFile,
             self
         )
+        self.fileFinderThread.finished.connect(self.fileFinderThread.deleteLater)
```

Comment:
Please update `_FileFinderThread` call, and connect `fetched` signal to `self._addFile()`.